src/PageBanner/PageBanner.scss

```diff
@@ -0,0 +1,47 @@
+.pgn__pageBanner-component {
```

Comment:
[design feedback] Should this class have some padding to account for the hover state on the icon button? As is, the hover state sits flush with the edges of this page banner container, which is a bit cleaner looking with a tiny bit of additional padding:

**Without padding**

![image](https://user-images.githubusercontent.com/2828721/129798114-4b9a6ef3-0e25-4ea6-8a1b-643b2bebe076.png)


**With padding**
Using `padding: map.get($spacers, 1);`  (equivalent to `.p-1` utility class)

![image](https://user-images.githubusercontent.com/2828721/129798267-dc8b30d2-85b4-438e-ad0a-78f74d3bd71d.png)

Reply:
For posterity, @ekangedx changed the design. We're now using the `inline` Icon Button rather than the `small` size, and because it's centered, it mimics that added padding suggestion.

Reply:
@ciduarte Great :) Thanks for bringing it to the Paragon Working Group for feedback. Using `inline` Icon Button is definitely preferable over adding explicit padding here.